src/controller/java/CHIPDeviceController-JNI.cpp

```diff
@@ -271,6 +272,11 @@ JNI_METHOD(void, pairDevice)(JNIEnv * env, jobject self, jlong handle, jlong dev
                                       .SetConnectionObject(reinterpret_cast<BLE_CONNECTION_OBJECT>(connObj))
                                       .SetBleLayer(&sBleLayer)
                                       .SetPeerAddress(Transport::PeerAddress::BLE());
+    if (csrNonce != nullptr)
+    {
+        JniByteArray jniCsrNonce(env, csrNonce);
+        params = params.SetCSRNonce(jniCsrNonce.byteSpan());
```

Comment:
Why does this assignment need to be here?

Reply:
It doesn't, I missed this. Will clean this up next time I touch this file...